File: 01_Algorithmic_Toolbox/week1_programming_challenges/1_sum_of_two_digits/aplusb.py
def SumofTwoDigits(d1, d2):
    # d1 and d2 should be integers
    answer = d1 + d2
    return answer

if __name__ == "__main__":
    a, b = map(int, input().split())
    print(SumofTwoDigits(a, b))

File: 01_Algorithmic_Toolbox/week1_programming_challenges/1_sum_of_two_digits/aplusb.R
SumofTwoDigits <- function(d1, d2){
    # d1 and d2 should be integers
    answer = d1 + d2
    return(answer)
}

n <- readline(prompt="")
a = as.integer(unlist(strsplit(n, " "))[1])
b = as.integer(unlist(strsplit(n, " "))[2])
print(SumofTwoDigits(a, b))
